examples/flutter_gallery/lib/welcome/step.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+const Color _kWelcomeBlue = Color(0xFF0175c2);
+
+abstract class WelcomeStep {
+  WelcomeStep({this.tickerProvider});
+  final TickerProvider tickerProvider;
+
+  String title();
+  String subtitle();
+
+  void animate({bool restart});
+  Widget imageWidget();
+  Widget contentWidget() => Material(
+        color: _kWelcomeBlue,
```

Comment:
indentation issue

Reply:
Fixed. That was run through dartfmt btw so maybe there is a bug? :)